packages/admin-ui/tsconfig.json

```diff
@@ -14,6 +14,6 @@
     "rootDir": "src",
     "skipLibCheck": true
   },
-  "include": ["src", "../vite.config.dev.ts"],
+  "include": ["src", "../vite.config.dev.ts", "../tsup.config.ts"],
```

Comment:
question: Since the root dir is src and we are including other files from the parent, will the dist  looks like
- dist
  - src
  - vite
  - tsup